### PR TITLE
misc: improve debug loading text wrap responsiveness

### LINKED ISSUE #25667: Debug 1.1 - Text in header of DebugTestLoadingContainer should wrap
**Current behavior**
Text used in the header of the `DebugTestLoadingContainer` overflows at smaller breakpoints.

**Desired behavior**
Text used in the header of the `DebugTestLoadingContainer` should wrap at smaller breakpoints.

### PR DESCRIPTION
**Pull request by ZachJW34**
- Closes https://github.com/cypress-io/cypress/issues/25667

### Additional details
Improves responsiveness of the Debug Page empty state header and remove full width header to align with Figma

### Steps to test
Run: https://github.com/cypress-io/cypress/blob/13dbd418a25a8e7b3d65ececbbd3c41dac220b33/packages/app/src/debug/empty/DebugEmptyStates.cy.tsx#L24

and check out the percy snaphots

### How has the user experience changed?
Screen Shot 2023-02-03 at 12 11 04 PM

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by mike-plummer** [comment]
@ZachJW34 Changelog entry?

**Comment by ZachJW34** [comment]
@mike-plummer done in [e6a78f8](https://github.com/cypress-io/cypress/pull/25703/commits/e6a78f8558a4e85a8cdc1a4c7f1580f4cf836d8f)

File: packages/app/src/debug/empty/DebugEmptyStates.cy.tsx [hunk 1]
```diff
@@ -33,6 +33,10 @@ describe('Debug page empty states', () => {
       cy.findByRole('link', { name: 'Learn about project setup in Cypress' }).should('have.attr', 'href', 'https://on.cypress.io/adding-new-project?utm_source=Binary%3A+Launchpad&utm_medium=Debug+Tab&utm_campaign=Learn+More')
 
       cy.percySnapshot()
+
+      cy.viewport(600, 600)
+
+      cy.percySnapshot('responsive')
```
Comment (astone123) [inline packages/app/src/debug/empty/DebugEmptyStates.cy.tsx:39]: I'm not seeing this snapshot in the [Percy build](https://percy.io/cypress-io/cypress/builds/24913506/) for some reason 🤔

**Review by astone123 — APPROVED** [review]
Looks good, might want to check why the new Percy snapshot isn't showing up 👍🏻

**Comment by ZachJW34** [comment]
@astone123 Hmm I'll give it a rerun, thought I saw it pop up on a previous run

Edit: https://percy.io/cypress-io/cypress/builds/24826035/changed/1386436804?browser=chrome&browser_ids=34&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=800&widths=600%2C800 shows the screenshot, might have been approved and so is no longer showing up

**Review by lmiller1990 — APPROVED** [review]

**Pull request merged**